### DISCUSSION TITLE
"Seed data" option is not available

### DISCUSSION THREAD

**Discussion by zolbayars** [planetscale/discussion#479] [Q&A] [answered]
When I try to create a new branch, the pop-up doesn't show the "Seed data" option ([as documented here](https://planetscale.com/docs/concepts/data-branching)). Is that beceuse I'm on a free plan? 🤔

![image](https://github.com/planetscale/discussion/assets/3589907/d6110588-c6f4-4f3a-97c9-659dba83de51)

**Comment by orware** [comment, marked as answer]
Hello @zolbayars,

That feature is noted on the documentation page you shared as being for the Team plan currently which is why you don't see it listed.
